File: nordeus_challenge_backend_spring_boot/src/main/java/com/mmileticc/nordeuschallenge/model/dto/ActiveBuffDTO.java
package com.mmileticc.nordeuschallenge.model.dto;

import com.mmileticc.nordeuschallenge.model.entity.enums.BuffAttribute;
import com.mmileticc.nordeuschallenge.model.entity.enums.CharacterSide;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ActiveBuffDTO {

    private CharacterSide targetSide;
    private BuffAttribute attribute;
    private int amount;
    private int remainingTurns;
}

File: Gauntlet of the Hero - GOTH/Assets/_Scripts/Models/Backend/ActiveBuffDto.cs
using Newtonsoft.Json;

namespace GauntletOfTheHero.Backend.Models
{
    [System.Serializable]
    public class ActiveBuffDto
    {
        [JsonProperty("targetSide")]
        public CharacterSide targetSide;

        [JsonProperty("attribute")]
        public BuffAttribute attribute;

        [JsonProperty("amount")]
        public int amount;

        [JsonProperty("remainingTurns")]
        public int remainingTurns;
    }
}